### ISSUE TITLE
locale file not working properly

### ISSUE THREAD

**Issue by MCLimited** [pop4959/LWCX#158]
**LWC Version (/lwc admin version):** 
2.2.7
**Server version (/version):** 
Paper spigot 1.16.5 (642)
**Short five word summary of issue:** 
Locale not working properly

**Detailed description of problem:**
LWC doesnt pickup all the locale strings. We translated the complete file, but it takes approximate 50% of the translations and 

This is what we filled in in the locale file:
protection.remove.protection.finalize=%dark_gray%[%gold%DMC Lock%dark_gray%] %gray%Sla op de beveiliging om het slot te verwijderen.

This is the translation shown in-game (default dutch translation)
https://i.imgur.com/mjzGiBm.png

How to resolve this?

**Comment by pop4959** [comment]
Can you provide the full locale file you're using, as well as core.yml.

**Comment by MCLimited** [comment]
core.yml
https://pastebin.com/AJeSqQt1

lwc_dmc.properties
https://pastebin.com/U7dsgNZR


Extra information:
I tried different versions from paper and it seemed like that it wasnt working in version 642 but did work in version 742. Maybe the bug was paper related? It is fixed for now but i wasn't 100% sure if it is paper related our plugin related. If its paper related just close the ticket :). Reported it to inform you. I uploaded the locale and core to pastebin in the above 2 links.

Kind regards
MC_Limited
